.github/actions/run-airbyte-ci/action.yml

```diff
@@ -115,7 +115,9 @@ runs:
         ls -la
         ls -la airbyte-python-cdk || echo "No airbyte-python-cdk directory"
         ls -laL ../airbyte-python-cdk || echo "No airbyte-python-cdk symlink"
-
+    - name: Install Java Environment
```

Comment:
For a followup: that would be cool to spare python connector only PR from installing the java env. We could rely on the `changes` output to check if any java connector was modified